src/visualizer/codeinformation/templates/radixSortTemplate.js

```diff
@@ -1,3 +1,9 @@
+/**
+ * Radix sort algorithm details.
+ *
```

Comment:
Is there a missing reference link?

Reply:
Perhaps i lost the link for this algorithm.